src/components/DefinitionEntry.js

```diff
@@ -71,16 +71,28 @@ export default class DefinitionEntry extends React.Component {
 
   renderHeadline(definition) {
     const { namespace, name, revision } = definition.coordinates
+    const locationUrl = this.getLocationURL(definition)
+    const revisionUrl = this.getRevisionUrl(definition)
     const namespaceText = namespace ? namespace + '/' : ''
     const sourceUrl = this.getSourceUrl(definition)
-    let revisionText = <span>&nbsp;&nbsp;&nbsp;{revision}</span>
+    let revisionText = (
+      <span>
+        &nbsp;&nbsp;&nbsp;
+        <a href={revisionUrl} target="_blank">
+          {revision}
+        </a>
+      </span>
+    )
+
     const location = get(definition, 'described.sourceLocation')
-    if (!location || (definition.coordinates.provider === location.provider && revision === location.revision))
+    if (location && (definition.coordinates.provider === location.provider && revision === location.revision))
       revisionText = ''
     return (
       <span>
-        {namespaceText}
-        {name}
+        <a href={locationUrl} target="_blank">
```

Comment:
what happens when there is no `locationUrl`?

Reply:
If there is no locationUrl the namespace and name is shown without a hyperlink. Along with that sometimes there isn't a namespace so just the name is shown.